src/utilities/util_verify.c

```diff
@@ -17,17 +17,12 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
     size_t size;
     int ch;
     char *config, *dump_offsets, *uri;
-    bool dump_address, dump_blocks, dump_layout, dump_pages, dump_history, hs_verify,
```

Comment:
I think there should be an obvious way to verify the metadata via the command line utility. It's probably possible to do that via -C with your changes, but I'd prefer something more explicit in here.

Reply:
I don't object to this. It is slightly unusual in that a verify of the metadata will effectively be a `wiredtiger_open` followed immediately by `connection->close`. That is okay. With the -C option, the user could do some other command.